src/crs/__init__.py

```diff
@@ -76,4 +76,25 @@
         axes_order_east_north_in_epsg_db=False,
         proj4text=None,
     ),
+    EPSG_5514: CRSTypeDef(
+        world_bbox=(
```

Comment:
Use values -951499.37 -1276279.09 -159365.31 -983013.08 (covers also Slovakia)

Reply:
You mean, those that are there already? :)